src/Component/subBreedList.tsx

```diff
@@ -0,0 +1,41 @@
+import axios from "axios";
+import { FunctionComponent, useEffect, useState } from "react";
+import { Link } from "react-router-dom";
+import LoadImg from './LoadImg'
+
+interface SubBreedListProps {
+    main:string
+}
+ 
+const SubBreedList: FunctionComponent<SubBreedListProps> = (props:SubBreedListProps) => {
+    let [subBreeds,setSubBreeds] = useState<string[]>([])
+    useEffect( () => {
+        axios
+        .get('https://dog.ceo/api/breed/'+props.main+'/list')
+        .then(res=>{
+            setSubBreeds(res.data.message);
+        })
+        .catch(err => {console.log(err);})
+    },[props.main])
+    if(subBreeds.length > 0)
+    return (<>
+        <h2 className="subBreeds">Sub Breeds for {props.main}</h2>
+        {
+            subBreeds.map( (subBreed,index) => (
+                <Link to={`${subBreed}`} key={index} >
+                    { <figure className={"subBreed-"+index}>

```

Comment:
How is this className naming indexing schema is used here?

Reply:
My idea was to create a complex grid layout for the breeds and subBreeds,
depending on the position they were placed.

So I added the _subBreed+index_ to differentiate them.
Now looking back I would add a function because the same convention is used in many places in my codebase